symfony/webpack-encore-bundle/2.0/webpack.config.js

```diff
@@ -37,7 +37,8 @@ Encore
      * https://symfony.com/doc/current/frontend.html#adding-more-features
      */
     .cleanupOutputBeforeBuild()
-    .enableBuildNotifications()
+    // Displays build status system notifications to the user
+    // .enableBuildNotifications()
```

Comment:
Just two newlines for visual separations with the rest, and that's good to me :)

```suggestion

    // Displays build status system notifications to the user
    // .enableBuildNotifications()

```

Reply:
Should the comment suggest installing the package this PR removes from the recipe dependencies?

Reply:
Yes, it's already the case, see https://github.com/symfony/webpack-encore/blob/main/lib/plugins/notifier.js#L30 and https://github.com/symfony/webpack-encore/blob/main/lib/features.js#L126-L132